traffic_ops/traffic_ops_golang/cdn/cdns.go

```diff
@@ -58,14 +58,6 @@ func (cdn TOCDN) GetKeyFieldsInfo() []api.KeyFieldInfo {
 	return []api.KeyFieldInfo{{"id", api.GetIntKey}}
 }
 
-//Implementation of the Identifier, Validator interface functions
-func (cdn TOCDN) GetKeys() (map[string]interface{}, bool) {
-	if cdn.ID == nil {
-		return map[string]interface{}{"id": 0}, false
-	}
-	return map[string]interface{}{"id": *cdn.ID}, true
-}
-
 func (cdn TOCDN) GetAuditName() string {
```

Comment:
This `func` accesses the `cdn.ID` field, which no longer has the right type, causing compilation issues.